client/src/pages/component/homeBody/Homebody.css

```diff
@@ -1,4 +1,88 @@
-.homebody{
-    text-align: center;
-    margin-top: 100px;
+.homebody-container {
+	background-image: url(../image/home-body.jpg);
+	background-repeat: no-repeat;
+	width: 100%;
+	height: 650px;
+	background-size: contain;
```

Comment:
<img width="693" alt="Screenshot 2022-11-18 at 14 57 32" src="https://user-images.githubusercontent.com/65277182/202734022-99e7b2fd-2421-4c4d-a345-31231d20adc1.png">

try backgroudn size cover.

Reply:
Cool but do you think `background-size: contain;` does the job better than `background-size: cover;`?